### DISCUSSION TITLE
Two Tracker code in One website

### DISCUSSION THREAD

**Discussion by aahonarmand** [umami-software/umami#624] [Q&A]
Hi

we have multiple websites and use UMAMI to track visitors. 

we use a SAME tracker code in All websites to ghather total vistors in one report.
and we add a second tracker code per website to get seperate reports.

but we findout that the second tracker is not working porperly and reports far less visitors.

so what is the problem?

thanks for your attention.

**Comment by mikecao** [comment]
You can't have two tracker scripts on a single site. You will probably need to write some functions to send the same stats to another website id using tracker functions, https://umami.is/docs/tracker-functions
